src/ansys/fluent/core/streaming_services/datamodel_event_streaming.py

```diff
@@ -66,12 +66,12 @@ def _process_streaming(self, id, stream_begin_method, started_evt, *args, **kwar
                         elif response.HasField("commandAttributeChangedEventResponse"):
                             value = response.commandAttributeChangedEventResponse.value
                             cb[1](_convert_variant_to_value(value))
-                        elif (
-                            response.HasField("modifiedEventResponse")
-                            or response.HasField("deletedEventResponse")
-                            or response.HasField("affectedEventResponse")
-                        ):
+                        elif response.HasField(
+                            "modifiedEventResponse"
+                        ) or response.HasField("affectedEventResponse"):
                             cb[1](cb[0])
+                        elif response.HasField("deletedEventResponse"):
+                            cb[1]()
```

Comment:
We should not capture the object which is going to be deleted, will update the PyFluent API.

Reply:
https://github.com/ansys/pyfluent/issues/3570